playbooks/ansible-test-cloud-integration-base/pre.yaml

```diff
@@ -0,0 +1,90 @@
+---
+- hosts: controller
+  tasks:
+#    - name: Add python38 support if needed
+#      block:
+#        - name: Ensure python3.8 is present
+#          become: true
+#          package:
+#            name: python3.8-dev
+#            state: present
+#      when:
+#        - ansible_os_family == "Debian"
+#        - ansible_test_python is version('3.8', '==')
+#
+#    - name: Select proper ansible_network_os
+#      set_fact:
+#        _network_os: "{{ hostvars[groups['appliance'][0]]['ansible_network_os'] }}"
+#      when: '"appliance" in groups'
+#
+#    - name: Select proper ansible_network_os
+#      set_fact:
+#        _network_os: "{{ hostvars[groups['openvswitch'][0]]['ansible_network_os'] }}"
+#      when: '"openvswitch" in groups'
+#
+#    - name: Setup tox role
+#      include_role:
+#        name: tox
+#      vars:
+#        tox_extra_args: "-vv -- ansible-playbook -v playbooks/ansible-test-network-integration-base/files/bootstrap-{{ _network_os }}.yaml"
```

Comment:
This is where the stuff above should move into. We run 2 nested bootstrap process for network, we should also do the same here.